pkg/proxy/proxy.go

```diff
@@ -96,15 +97,20 @@ type Proxy struct {
 	// Datapath updater for installing and removing proxy rules for a single
 	// proxy port
 	datapathUpdater DatapathUpdater
+
+	// IPCache is used for tracking IP->Identity mappings and propagating
+	// them to the proxy via NPHDS in the cases described
+	ipcache *ipcache.IPCache
```

Comment:
I don't see this new `ipcache` member being used in this package (maybe it will be used in later commits?).

Reply:
Correct, I moved it to NPHDS and never got rid of this one. Removed, good spotting.